src/Parser/DeathKnight/Shared/getDamageBonusStacked.js

```diff
@@ -0,0 +1,8 @@
+export default function getDamageBonusStacked(event, increase, stacks) {
```

Comment:
You should import this from core instead of duplicating it here just for dks:
https://github.com/WoWAnalyzer/WoWAnalyzer/blob/master/src/Parser/Core/calculateEffectiveDamageStacked.js